packages/legacy/core/App/localization/en/index.ts

```diff
@@ -237,6 +237,7 @@ const translation = {
     "PinChangeSuccessTitle": "Successfully changed your PIN",
     "PinChangeSuccessMessage": "Your PIN was successsfully changed. You can now log in with your new PIN.",
     "EnterOldPINTitle": "Enter your old PIN",
+    "EnterOldPIN": "Enter current PIN",
```

Comment:
Should we just use the existing entry above this? There may be confusion about "old" vs "current"